citest/base/global_journal.py

```diff
@@ -58,8 +58,8 @@ def new_global_journal_with_path(path, **metadata):
       atexit.register(_atexit_handler)
       _added_atexit = True
 
-    journal_file = open(path, 'w')
-    os.fchmod(journal_file.fileno(), 0600)  # Protect sensitive data.
+    journal_file = open(path, 'wb')
+    os.fchmod(journal_file.fileno(), 0o600)  # Protect sensitive data.
```

Comment:
Should we use `stat` constants instead of the raw file permissions? https://stackoverflow.com/questions/16249440/changing-file-permission-in-python